feature: align cover svg to viewer

wolasss opened the pull request:
These changes allow to call:

`viewer.fitToViewer(ALIGN_COVER, ALIGN_COVER)` so that the svg will always take the whole viewport and match the viewport by the short edge. 

Please review and let me know if there is something else I must add

wolasss commented:
It fixes https://github.com/chrvadala/react-svg-pan-zoom/issues/29

wolasss commented:
@chrvadala hey, any chances this will be merged soon? :)

chrvadala commented on docs/documentation.md:
```diff
@@ -107,8 +107,8 @@ import {
-  
-  //functions to manipulate the state. The signature is quite similar to component methods, expect for the first parameter that is the current value. These are usefull only for some advanced integrations (e.g. reducers). 
+
+  //functions to manipulate the state. The signature is quite similar to component methods, expect for the first parameter that is the current value. These are usefull only for some advanced integrations (e.g. reducers).
```
This PR changes a lot of blank line. Can you clean this file and submit only the changed text?

chrvadala commented on package-lock.json:
```diff
@@ -0,0 +1,19103 @@
+{
```
Can you remove this file from PR? It might overlap with `yarn.lock`

chrvadala commented:
> Hi, thanks for your contributions.
> I added few feedbacks inline; moreover, can you provide some unit tests?
> 
> Finally, let me to suggest you to align your PR with the add-test branch https://github.com/chrvadala/react-svg-pan-zoom/tree/add-tests, because I'll merge it in the next few days

quick update: add-tests is merged! you can sync your branch with master

wolasss commented:
Ok, thanks, I'll address all the issues after the weekend

wolasss commented:
@chrvadala I added some unit tests - for two scenarios where rect > viewport  and the opposite. Both cases seem to be working fine. I also removed the package-lock.json and removed the blank lines.

chrvadala commented:
Released with version 3.8.0
Thanks for your contribution, I added you in contributors list
